ppx-driver/gen_js_api_ppx_driver.ml

```diff
@@ -44,22 +12,16 @@ let () =
     Ppxlib.Driver.enable_location_check ()
   );
   Gen_js_api_ppx.mark_as_handled_manually := (fun attribute ->
-      let attribute = To_ppxlib.copy_attribute attribute in
       Ppxlib.Attribute.mark_as_handled_manually attribute);
   let mapper_for_sig =
-    To_ppxlib.copy_mapper
-      (Gen_js_api_ppx.mark_attributes_as_used Gen_js_api_ppx.mapper)
```

Comment:
I think this is wrong.

Reply:
Actually it seems that the recursive use of the "Gen_js_api_ppx.mapper" was not used.